README.md

```diff
@@ -11,6 +11,10 @@ your application are configured.
 Once you have a Compose file, you can create and start your application with a
 single command: `docker compose up`.
 
+# About v2 update compatibility breaks
+
+Docker Compose v2 is a major version bump release of Docker compose. It has been completely rewriten from scratch in Golang (v1 was in Python). The installation instructions for Compose v2 differ from v1. v2 is not a standalone binary anymore and installation scripts will have to be adjusted. Some commands are different. For a smooth transition from legacy docker-compose 1.xx, please consider installing [compose-switch](https://github.com/docker/compose-switch) to translate `docker-compose ...` commands into Compose v2's `docker compose .... `. Also check v2's `--compatibility` flag.
```

Comment:
rest of the documentation uses uppercase "V1" and "V2"
please place the compose-switch note as a separate paragraph to make it bit more visible to readers

Reply:
done